output of running make in `termbox_next` WITHOUT the suggested change (and none of my changes) is as follows. it is identical to the output WITH your suggested change (verified by diff`)

```
❯ make
building source object obj/src/termbox.o
building source object obj/src/input.o
building source object obj/src/memstream.o
building source object obj/src/ringbuffer.o
building source object obj/src/term.o
building source object obj/src/utf8.o
compiling bin/termbox.a
ar: creating archive bin/termbox.a
a - obj/src/termbox.o
a - obj/src/input.o
a - obj/src/memstream.o
a - obj/src/ringbuffer.o
a - obj/src/term.o
a - obj/src/utf8.o
```

HOWEVER re [the issue in jermbox](https://github.com/MorganPeterson/jermbox/issues/6) is still present after replacing the contents of its `termbox_next` folder with the contents of this repo with the makefile tweak.   Tested manually and via repointing the submodule. 


```
...
compiling termbox_next/src/memstream.c to build/termbox_next___src___memstream.o...
generating meta file build/jermbox.meta.janet...
termbox_next/src/termbox.c:115:13: error: use of undeclared identifier 'SIGWINCH'
  sigaction(SIGWINCH, &sa, 0);
            ^
1 error generated.
error: command failed with non-zero exit code 1
compiling src/jermbox.c to build/src___jermbox.static.o...
...
```

--- a/makefile
+++ b/makefile
@@ -7,6 +7,10 @@ ifeq ($(OS),Linux)
 	FLAGS+=-D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700
 endif
 
+ifeq ($(OS),Darwin)
+    FLAGS += -g -D_DARWIN_C_SOURCE
+endif
+
 INCLUDED=/usr/include
 LIBD=/usr/lib
 

--- a/src/termbox.c
+++ b/src/termbox.c
@@ -25,6 +25,11 @@
 #define LAST_ATTR_INIT 0xFFFFFFFF
 #define ENOUGH_DATA_FOR_INPUT_PARSING 128
 
+#ifndef SIGWINCH
+#define SIGWINCH 28  /* window size changes */
+#define SIGINFO 29 /* information request */
+#endif
+
 struct cellbuf
 {
   int width;

